dev-requirements.txt

```diff
@@ -8,6 +8,8 @@ isort==4.3.4
 jupyter==1.0.0
 matplotlib==3.0.2; python_version>'3.4'
 matplotlib==2.2.3; python_version<'3'
+nbconvert==5.4.1
```

Comment:
jupyter==1.0.0 installs nbconvert

Reply:
yes but nbsphinx does not want nbconvert==5.4.0
https://github.com/spatialaudio/nbsphinx/blob/master/setup.py#L18